website/docs/getting-started.md

```diff
@@ -4,191 +4,184 @@
 
 1. Deploy ORC to your Kubernetes cluster
 1. Create a Kubernetes secret containing a `clouds.yaml`
-1. Create an OpenStackCloud object pointing to that `clouds.yaml`
 1. Deploy your OpenStack infrastructure as Kubernetes custom resources
 
 ## Deploy ORC to your Kubernetes cluster
 
-From the git repository, run:
+From a fresh checkout of the git repository, from the `main` branch, run:
 
 ```sh
-make deploy IMG=quay.io/orc/openstack-resource-controller
+make deploy IMG=quay.io/orc/openstack-resource-controller:branch-main
 ```
 
 ## Create a Kubernetes secret containing a `clouds.yaml`
 
 ```sh
 kubectl create secret generic openstack-clouds \
-    --from-file=${XDG_CONFIG_HOME}/openstack/clouds.yaml
+    --from-file=${XDG_CONFIG_HOME:-~/.config}/openstack/clouds.yaml
 ```
 
 **Note:**
 > The command above will upload your entire `clouds.yaml` to your Kubernetes
 > cluster! If that is not appropriate, you may want to upload a slimmed version
 > of it.
```

Comment:
e.g. The user may break themself in following this instruction in a manner which isn't immediately obvious.